### ISSUE TITLE
[Feature Request] Being able to jump to a later point in a VOD and all the others follow

### ISSUE THREAD

**Issue by PikaDude** [henryperson/twitchmultivod#9]
When the VODs are all synced and playing along with each other, trying to fast forward one of them doesn't fast forward the rest of them. This means jumping to specific points in the session means I need to manually calculate it for each VOD and jump manually, kind of deceiving the point of having them in sync.

**Comment by henryperson** [comment]
Yep, this is definitely a feature I'd like to implement. Thanks for pointing it out, hoping I can get to it soon. Main problem in my way is that twitch does not make it easy to detect when you fast forward, though I think there are probably workarounds to it.

In the meantime, unless I'm missing something you shouldn't need to manually calculate differences and jump. If you have your videos synced, and you jump ahead in a video, you can just hit "Sync To This" for whichever video you jumped in and the rest will sync to it. Not an ideal user experience, but at least it's better than manually doing it :)
